### DISCUSSION TITLE
Very long access time to web interface of AdGuard Home :

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4709] [General]
Hello, 

For some time when I connect to the web interface of AdGuard Home, I notice a huge slowness indeed, the loading is very long or even impossible: it loads in a vacuum !

 I tried on all my devices : my phone, my personal pc, my son's pc ... it's always the same !

 I am constantly forced to restart my Raspberry in SSH in order to boost AdGuard Home ...

 Is it a bug ?

**Comment by fernvenue** [comment]
I don't think so, bug must be reproducible. But maybe you can do something about it:

- Check other applications on your pi if they have the same issue.
- Use the dev tool on your browser (`F12`) to see what's going on of your connection with your pi.
- Enable verbose log using `-v, --verbose` to see what's happen.

If restarting the service temporarily solves the problem, and you don't want to spend too much time on it, maybe you can just use crontab or systemd timer to automatically restart your AdGuardHome every single day or week.

All in all, there are so many possibilities, hope you can quickly figure it out :)

**Reply by ghost** [reply]
Thank you for your reply ! 

 In fact, it's just that the Pi was too far from my internet box...
